tests/sles4sap/forkbomb.pm

```diff
@@ -27,6 +28,34 @@ sub run {
     my $sid = get_required_var('INSTANCE_SID');
     my $instance_id = get_required_var('INSTANCE_ID');
     my $sapadm = $self->set_sap_info($sid, $instance_id);
+
+    my $package_version = script_output "rpm -q --qf '%{VERSION}' saptune";
+    record_info('saptune version', $package_version);
+    my $note_id = '2578899';
+    my $conf_kernel_pid_max_old = script_output("grep kernel.pid_max /usr/share/saptune/notes/$note_id", proceed_on_failure => 1);
+    record_info('Old conf value', $conf_kernel_pid_max_old);
+    my $os_kernel_pid_max_old = script_output('sysctl -a | grep kernel.pid_max', proceed_on_failure => 1);
+    record_info('Old kernel.pid_max', $os_kernel_pid_max_old);
+
+    # Reset kernel.pid_max according to the changes of SAP Note 2578899
+    # See https://github.com/SUSE/saptune/commit/a81168759681a05437e60429de0ddc1e19c703f4
+    # Or see https://github.com/SUSE/saptune/releases/tag/3.1.3 for more details
+    if (package_version_cmp($package_version, '3.1.3') >= 0) {
+        my $pid_max = get_var('KERNEL_PID_MAX', 40000);
```

Comment:
May I suggest to use a name that cite `OPENQA` or generally speakig refer to the fact that it is the max value openQA machine can support? We should try to find a name that clearly distinguish from `pid_max` in saptune notes and `pid_max` configured at OS level